backend/app/Http/Response/ApiResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Response;
+
+use App\Contracts\ApiException as ApiExceptionContract;
+use App\Contracts\ApiResponse as ApiResponseContract;
+use Illuminate\Http\JsonResponse;
+
+abstract class ApiResponse extends JsonResponse
+{
+    public static function success(ApiResponseContract $response): self
```

Comment:
toArray set in data, like this

[
     "data" => $response->toArray(),
     "meta" => [] // or optional parameter array $meta
]